src/i18n/en.json

```diff
@@ -169,6 +169,11 @@
     "transaction.group.createRootMotion.title": "Create root motion",
     "transaction.group.createRootMotion.description": "Create root motion",
 
+    "transaction.VotingReputationClient.createRootMotion.title": "Create root motion",
+    "transaction.VotingReputationClient.createRootMotion.description": "Create root motion",
+    "transaction.group.createRootMotion.title": "Create root motion",
+    "transaction.group.createRootMotion.description": "Create root motion",
```

Comment:
Why is this in here?

Especially since the exact same lines are duplicated above. I assume this is a rebasing error

Reply:
That's correct! Encountered this exact same annoying error on other rebases :facepalm: 

Must have slipped my radar here